bucket/poedit.json

```diff
@@ -0,0 +1,26 @@
+{
+    "version": "2.2.3",
+    "homepage": "https://poedit.net/",
+    "description": "Gettext translations editor.",
+    "license": {
+        "identifier": "Shareware,MIT",
+        "url": "https://poedit.net/download"
```

Comment:
Shareware is not mentioned anywhere on this page

Reply:
> Up-to-date source code of the Open Source version of Poedit (sans the Pro features included in the above binaries) is available at GitHub under the terms of the MIT license. 

It has a pro edition. Is it a shareware?

Reply:
Where is said you can distribute pro edition?

Reply:
The pro edition can be bought and upgrade in app.

Reply:
Then how could it be shareware? 

> Provided free of charge to users, who are allowed and encouraged to make and share copies of the program

There is not mentioned sharing/distributing of program.

Reply:
So freeware?

Reply:
https://github.com/vslavik/poedit


> License
> =========
> Poedit is released under the MIT license and you're free to do whatever you want with it and its source code (well, almost :-) -- see the license text).
> See the COPYING file for details on the program's licensing and the icons/README file for details on the icons.
> Windows and macOS versions contain GNU gettext binaries. They are distributed under the GNU General Public License and their source code is available from http://www.gnu.org/software/gettext. If you have difficulties getting them
from there, email me for a copy of the sources.
>>>